diff --git a/es/index.d.ts b/es/index.d.ts
index 7a11d241ac7491029343a36f62a3d32b13757855..8cd3cab0e3357d5bc4ddc23f76fe92cd67d55b4d 100644
--- a/es/index.d.ts
+++ b/es/index.d.ts
@@ -22791,7 +22791,7 @@ declare type TableGridInstance = InstanceType<typeof TableGrid> & UnwrapRef<{
      */
     resetAfterRowIndex: ResetAfterIndex;
 }>;
-declare interface TableHeader extends ComponentInternalInstance {
+export declare interface TableHeader extends ComponentInternalInstance {
     state: {
         onColumnsChange: (layout: TableLayout<any>) => void;
         onScrollableChange: (layout: TableLayout<any>) => void;
@@ -24693,7 +24693,7 @@ export declare interface TreeComponentProps {
     icon: string | Component;
 }
 export declare type TreeData = TreeNodeData[];
-declare interface TreeData_2 extends TreeNode {
+export declare interface TreeData_2 extends TreeNode {
     children?: string[];
     lazy?: boolean;
     loaded?: boolean;
@@ -24755,7 +24755,7 @@ declare interface TreeOptionProps_2 {
         [key: string]: boolean;
     };
 }
-declare interface TreeProps {
+export declare interface TreeProps {
     hasChildren?: string;
     children?: string;
     checkStrictly?: boolean;
